package.json

```diff
@@ -24,7 +24,7 @@
     "build:server": "babel --config babel.config.js -x .js,.jsx -d dist/server src/client src/server/page.html.jsx",
     "docker:build": "docker build -f .circleci/Dockerfile -t 711655675495.dkr.ecr.us-east-1.amazonaws.com/react-pwa:latest .",
     "docker:run": "docker run -i -d -t --privileged --name=react-pwa --hostname=react-pwa -p 3000:3000 711655675495.dkr.ecr.us-east-1.amazonaws.com/react-pwa",
-    "lint": "eslint src/**/*.js src/**/*.jsx",
+    "lint": "eslint --ext .js --ext .jsx src",
```

Comment:
should be able to be `--ext .js,.jsx`, maybe more readable

Reply:
Good suggestion, have fixed.